platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/ConcurrentArray.java

```diff
@@ -272,9 +273,38 @@ void add(T element) {
     }
 
     /**
-     * Get a sorted stream of all elements. The comparator will be used to sort the elements.
-     * If the comparator is null, then the natural ordering of the elements will be used.
-     * This method can only be called on immutable instances. It may be called concurrently.
+     * Get a stream of all elements in this concurrent array.
+     *
+     * <p>This method can only be called on immutable instances. It may be called concurrently.
+     *
+     * @return A non-null stream over all elements
+     * @throws IllegalStateException
+     * 		If this instance is not immutable
+     */
+    Stream<T> stream() {
+        return stream(false, null);
+    }
+
+    /**
+     * Get a stream of all elements in this concurrent array. The elements will be sorted using the
+     * provided comparator, if not null, or in natural order.
+     *
+     * <p>This method can only be called on immutable instances. It may be called concurrently.
+     *
+     * @return A non-null sorted stream over all elements
+     * @throws IllegalStateException
+     * 		If this instance is not immutable
+     */
+    Stream<T> stream(@Nullable final Comparator<T> comparator) {
```

Comment:
Is this method used outside tests?

Reply:
This method is removed in the latest fix version.